meta-lmp-bsp/dynamic-layers/tegra/recipes-kernel/linux/linux-tegra-rt_5.10.bb

```diff
@@ -0,0 +1,6 @@
+require recipes-kernel/linux/linux-tegra_5.10.bb
+
+# Use BSP with rt-patches applied
+SRC_REPO = "github.com/foundriesio/linux.git;protocol=https"
+SRCBRANCH = "oe4t-patches${LINUX_VERSION_EXTENSION}-rt"
```

Comment:
It is a good idea to have the variable `LINUX_VERSION_EXTENSION` here and thus follow the kernel upstream branch.